src/api/java/com/minecolonies/api/util/InventoryUtils.java

```diff
@@ -49,7 +49,7 @@
      */
     private InventoryUtils()
     {
-        /*
```

Comment:
these lines aren't javadoc. they should remain as they were. IDE even gives a warning about it